src/pipeline_eval.py

```diff
@@ -7,6 +7,7 @@ def create_pipeline(  # noqa: PLR0913
     pipeline_dir: str = "./data-dir",
     embed_model_provider: str = "huggingface",
     embed_model: str = "all-MiniLM-L6-v2",
+    api_keys: dict = {},
```

Comment:
```suggestion
    embed_api_key: dict = {},
```

Can we rename this to make clear it's for the embedding component?

Reply:
sure!